src/alfasim_score/converter/alfacase/base_operation.py

```diff
@@ -50,224 +46,172 @@
-                            FLUID_OIL: Array([0.0], FRACTION_UNIT),
-                            FLUID_WATER: Array([0.0], FRACTION_UNIT),
-                        },
-                    ),
+            ),
+        )
+
+    def create_well_initial_temperatures(
+        self, top_temperature: Scalar, bottom_temperature: Scalar
+    ) -> InitialTemperaturesDescription:
+        """Create the initial temperatures description."""
+        well_length = self.alfacase_converter.get_position_in_well(
+            self.score_input.read_general_data()["final_md"]
+        )
+        return InitialTemperaturesDescription(
+            position_input_type=TableInputType.length,
+            table_length=TemperaturesContainerDescription(
+                positions=Array([0.0, well_length.GetValue()], LENGTH_UNIT),
+                temperatures=Array(
+                    [
+                        top_temperature.GetValue(TEMPERATURE_UNIT),
+                        bottom_temperature.GetValue(TEMPERATURE_UNIT),
+                    ],
+                    TEMPERATURE_UNIT,
                 ),
-                velocities=InitialVelocitiesDescription(
-                    position_input_type=TableInputType.length,
-                    table_length=VelocitiesContainerDescription(
-                        positions=Array([0.0], LENGTH_UNIT),
-                        velocities={
-                            FLUID_GAS: Array([0.0], VELOCITY_UNIT),
-                            FLUID_OIL: Array([0.0], VELOCITY_UNIT),
-                            FLUID_WATER: Array([0.0], VELOCITY_UNIT),
-                        },
-                    ),
+            ),
+        )
+
+    def configure_pvt_model(self, alfacase: CaseDescription) -> None:
+        """Configure the pvt fluid for the model."""
+        pass
+
+    def configure_outputs(self, alfacase: CaseDescription) -> None:
+        """Configure the outputs for the case."""
+        alfacase.outputs = CaseOutputDescription(
+            trends=TrendsOutputDescription(
+                global_trends=[GlobalTrendDescription(curve_names=["timestep"])]
+            ),
+            profiles=[
+                ProfileOutputDescription(
+                    curve_names=self.default_output_profiles,
+                    location=OutputAttachmentLocation.Main,
+                    element_name=WELLBORE_NAME,
+                )
+            ],
+        )
+
+    def configure_well_initial_conditions(self, alfacase: CaseDescription) -> None:
+        """Configure the well initial conditions with default values."""
+        alfacase.wells[0].initial_conditions = InitialConditionsDescription(
+            volume_fractions=InitialVolumeFractionsDescription(
+                position_input_type=TableInputType.length,
+                table_length=VolumeFractionsContainerDescription(
+                    positions=Array([0.0], LENGTH_UNIT),
+                    fractions={
```

Comment:
Are you sure that it is the same for both production and injector wells?

Reply:
Those fractions are arbitrary because we don't have any tip of them in the SCORE file.

Reply:
Fixed